guides/advanced_guides/known_limitations.md

```diff
@@ -0,0 +1,31 @@
+# Known limitations
+
+MeiliSearch has known limitations, some of which may be addressed in future versions.
+
+Some of these limitations have been made by MeiliSearch developers for relevacy and/or performance while others limitations are enforced by LMDB, the key-value store used by MeiliSearch.
+
+## Design limitations
+
+### Database size
+
+MeiliSearch uses two databases: the first one for storage and the second one for updates.
+On launch LMDB needs to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory.
+
+The maxime database size is by default __100GiB__ for each databases. This size can me modified using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration guide](/guides/advanced_guides/configuration.md#max-mdb-size).
+
+### Number of indexes
+
+You can create __up to 200 indexes__ in MeiliSearch. This limit has been hard set for performance reasons.
+
+### Maximum words per attribute
+
+A maximum of __1000 words par attribute__ can be indexed. This means that if an attribute contains more than 1000 words, only the first 1000 words will be indexed and the rest will be silently ignored.
```

Comment:
I would keep key/value terms.
```suggestion
A maximum of __1000 values attached to one attribute__ is set. This means that if an attribute contains more than 1000 values, only the first 1000 values will be indexed and the rest will be silently ignored.
```

Reply:
I think when thomas uses the word "indexes", he does not say "you can only have 1000 words per attribute" but "MeiliSearch can only **index** 1000 words per attributes". 
After which we explain what this imply. 

Also, there is a difference between values and word. A word is a group of chars inside a string separated by a white space: 
`wordOne wordtwo ..`

Whereas a value is the whole data linked to an attributes: 

```json
{
 "attribute": value
}
```

So, 1000 values for one attribute is technically not possible because an attribute can only have one value.

Reply:
Oh I see!

Reply:
I updated my suggestion. However, a longer explanation would be welcome.

Reply:
Do you mean I should explain the problem better or this part in the article should be explained more in depth?

I think all those limitation should be added to the relative guides. As for example this information should be added to the `documents` guide. Where in the context of the guide it makes more sense. 

Also, we could add a sort of definition in the glossary for the word "indexation".
What do you think of that?

```
In meilisearch, documents are __indexed__. Indexing is the process of saving data in a database in a prefered way to fit your purpose. In MeilISearch's case, the purpose is to optimise word-based search. Because of that, in MeiliSearch the indexation is centered arround words. 
```

Reply:
I agree about adding limitations to each relative guide. And yes it was about clarifying "indexation" term so that would be a good idea to add it to the glossary.